client/404.html

```diff
@@ -0,0 +1 @@
+We're not here ,buddy, try us at port 2000
```

Comment:
Actually is port 3000. If you're not accessing the correct port you're not receiving this message anyways.